DataSetExplorer/DataSetSerializer/AnnotatorData/annotator_data.csv

```diff
@@ -0,0 +1,4 @@
+﻿id;yearsOfExperience;ranking
+1;6;1
```

Comment:
This type of data shouldn't be in the source code unless it's test data (and while we're at it, the tests should also have this data somewhere, along with the BK test data).

For our purposes you can hard code it in Program.cs until we refactor it but don't introduce this file to the repo.